pkg/constants/constants.go

```diff
@@ -4,8 +4,9 @@ const (
 	NamespaceDefault         = "default"
 	NamespaceHarvesterSystem = "harvester-system"
 
-	FieldProviderKubeConfig  = "kubeconfig"
-	FieldProviderKubeContext = "kubecontext"
+	FieldProviderKubeConfig       = "kubeconfig"
+	FieldProviderKubeContext      = "kubecontext"
+	FieldProviderKubeConfigBase64 = "kubeconfig_base64"
```

Comment:
It doesn't look like this is being used.

Reply:
+1, we should remove this.

Reply:
Removed. Good catch.